gateway/proxy/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9.16-slim-buster
+
+WORKDIR /usr/src/app
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+USER 0:123
+RUN apt-get -y update
+RUN apt-get install -y iptables
+RUN apt-get install -y tcpdump
+RUN pip install headerparser
+RUN pip install certifi
+RUN pip install requests
+COPY gateway/proxy/proxy .
```

Comment:
Similar to my previous comment. I would separate this project from `gateway`.

Reply:
OK